integrations/crowdnode/src/main/res/layout/dialog_staking.xml

```diff
@@ -14,55 +14,248 @@
-        app:layout_constraintEnd_toEndOf="parent"
-        app:srcCompat="@drawable/staking_image" />
-
-    <TextView
-        android:id="@+id/staking_title"
-        style="@style/Headline5"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/crowdnode_staking_info_title"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/staking_image" />
-
-    <TextView
-        android:id="@+id/staking_message"
-        style="@style/Body2"
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/scrollbar"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginHorizontal="20dp"
-        android:layout_marginTop="10dp"
-        android:gravity="center"
-        android:textAlignment="gravity"
-        android:text="@string/crowdnode_staking_info_message"
-        app:layout_constraintTop_toBottomOf="@+id/staking_title" />
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        android:layout_height="match_parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageButton
+                android:id="@+id/collapse_button"
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:layout_margin="10dp"
+                android:background="?attr/selectableItemBackgroundBorderless"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/ic_popup_close" />
+
+            <ImageView
+                android:id="@+id/staking_image"
+                android:layout_width="240dp"
+                android:layout_height="240dp"
+                android:layout_marginTop="20dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/staking_image" />
+
+            <TextView
+                android:id="@+id/staking_title"
+                style="@style/Headline5"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="20dp"
+                android:text="@string/crowdnode_staking_info_title"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/staking_image" />
+
+            <TextView
+                android:id="@+id/staking_message_one"
+                style="@style/Body2"
```

Comment:
The secondary text on this screen seems to have the wrong color. I should be `#525C66`, so `Body2.Secondary` should be used.

Reply:
Fixed.